masschallenge/app-allocator-scripts
# AC-5421: First draft of the OrderedQueues heuristic

To test:
- `python3 allocate.py example.csv ordered_queues`

Run the results through analyze.py.  See the best results yet.

"missed Home Program Reads: 1" is due to the startup that has a program value of "Unknown Program".  It would be nice if the analyzer made this a bit clearer.

This PR does not complete this ticket since the universal queues and the reassignment queues are not yet implemented.
